src/Traits/Statable.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Sebdesign\SM\Traits;
+
+use SM\Factory\FactoryInterface;
+use SM\StateMachine\StateMachine;
+use Sebdesign\SM\Models\StateHistory;
+
+/**
+ * Trait Statable.
+ */
+trait Statable
+{
+    /**
+     * @var StateMachine
+     */
+    protected $stateMachine;
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    public function history()
```

Comment:
Is `history` not a bit too generic.
In case that someone has a revision package or something else, it's possible that the name is already in use.

Maybe it makes sense to rename it to `stateHistory`, then you can make sure that the name is unique (hopefully).

Reply:
I agree with that, `stateHistory` sounds better.